api-reference/10 UI Components/dxBarGauge/1 Configuration/resolveLabelOverlapping.md

```diff
@@ -6,6 +6,16 @@ default: 'hide'
 ---
 ---
 ##### shortDescription
-Specifies how the UI component should behave when [bar labels](/api-reference/10%20UI%20Components/dxBarGauge/1%20Configuration/label '/Documentation/ApiReference/UI_Components/dxBarGauge/Configuration/label/') overlap: hide certain labels or leave them overlapped.
+Specifies how the UI component should behave when [bar labels](/api-reference/10%20UI%20Components/dxBarGauge/1%20Configuration/label '/Documentation/ApiReference/UI_Components/dxBarGauge/Configuration/label/') overlap.
 
 ---
+If a BarGauge displays several values close to each other, the corresponding labels may overlap. If you want to avoid overlapped labels, use the **resolveLabelOverlapping** property to specify how the gauge adjusts label position or visibility. The following values are available: 
+
+- *'hide'*    
+The gauge hides all labels with the smallest values except the first one.
```

Comment:
When you say "except the first one", do you mean that all values that are _smaller than the first label's value_ are hidden? Please clarify.

Reply:
I wanted to say that all smaller labels are hidden except the label with the smallest value. Like if you had [2, 3, 3, 3, 4, 5] labels, after the algorithm does its work it should be something like [2, 4, 5]. So all the smallest except the smallest of them all 😄

Reply:
Thanks, that's totally better!